my-app/src/App.js

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
-import logo from './logo.svg';
+import Header from './Header.js';
+import Image from './Image.js';
+import Footer from './Footer.js';
+import Recipe from './Recipe.js';
+import Ingredients from './Ingredients.js';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+
+export default class AllPutTogether extends React.Component {
+  render() {
+    return(
+        <div>
```

Comment:
you don't actually need to add these semantic header, main, footer elements. you can just pass in the components you made like so: 

<div>
<Header />
<Image />
<Recipe />
....
</div>

Reply:
Cool. I'll look into it. Thank you!